### DISCUSSION TITLE
Standup (Thursday, April 21, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#18] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Sprint planning
 - Finished #10 
 - Began #14 

**What are your priorities for the next 24 hours?**:
 - Continue with #14 
 - Read about Github Actions

**Blockers**:
 - None

**Shoutouts**:
 - @CristianCazares @antonieto @BrandonMagana @DavBot02 for meeting at the apartment and being a cool team

**Comment by paotg123** [comment]
What are your priorities for the next 24 hours?
Continuar con #13 
Establecer diferencias entre perfil de postulante y de administrador de empresa y como se redactaran en #13

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Started #12 (Only a template so far)

**What are your priorities for the next 24 hours?**:
 - Continue with #12 (Write down introduction)

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19 @paotg123 @antonieto @BrandonMagana @DavBot02 [This is the document link.](https://docs.google.com/document/d/17ytxyNExtEWtkyx0g7fc8vjH3bjLRU86qwT_7jVbfU4/edit). You can also find it on the #12 issue.

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #9 

**What are your priorities for the next 24 hours?**:
 - Start #11 

**Blockers**:
 - None 

**Shoutouts**:
 - @CristianCazares for his first in person meeting

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished  #8 I was assigned to
 - Finished #9 
 
**What are your priorities for the next 24 hours?**:
 -  Start working on #13 (specially on FR)

**Blockers**:
 - Don't know what are the NFR that our client is expecting for this project 

**Shoutouts**:
 - @CarlosCRG19 @paotg123 @antonieto @CristianCazares @DavBot02  for doing such a great teamwork

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:

- Finished #7 
- Sarted #15 
- Merged PDFs and turned in [👷🏻‍♂️ User_Persona_Escenarios_Historias](https://experiencia21.tec.mx/courses/250537/assignments/8577750/submissions/120344) and [Diagramas de uso y de actividad](https://experiencia21.tec.mx/courses/250537/assignments/8581139/submissions/120344) 

**What are your priorities for the next 24 hours?**:
 - Continue with #15

**Blockers**:
 - None atm

**Shoutouts**:
 - @paotg123 for getting the pizza! 🍕
